src/vm/errors/runner_errors.rs

```diff
@@ -73,7 +73,7 @@ pub enum RunnerError {
     BuiltinSegmentNameCollision(&'static str),
     #[error("Error while finalizing segments: {0}")]
     FinalizeSegements(MemoryError),
-    #[error("finalize_segments called but proof_mode is not enabled")]
+    #[error("Finalize_segments called but proof_mode is not enabled")]
```

Comment:
```suggestion
    #[error("finalize_segments called but proof_mode is not enabled")]
```

The function name is lower case.

Reply:
fixed